src/Inventory/Request.php

```diff
@@ -114,8 +114,13 @@ protected function handleTask($task): array
             case self::INVENT_TASK:
                 return $this->handleInventoryTask();
             break;
+            case self::NETDISCOVERY_TASK:
```

Comment:
I do not agree, if there is no plugin that support those tasks; GLPI will just answer "nothing"

Reply:
This is not true. handleTask() is just called during agent CONTACT requests to handle tasks hooks. GLPI will answer as usual and the contact answer won't be updated with any handled tasks related information. The inventory hook is actually only used by the database plugin to notify it knows some database credentials.